go-controller/pkg/ovn/pods.go

```diff
@@ -3,17 +3,25 @@ package ovn
 import (
 	"fmt"
 	"net"
+	"net/http"
 	"strings"
 	"time"
 
+	hotypes "github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/types"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/metrics"
 	util "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 	kapi "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/klog"
 )
 
+func isNotFoundError(err error) bool {
```

Comment:
@rcarrillocruz instead of open-coding this, should we use `k8s.io/apimachinery/pkg/api/errors` IsNotFound() ?

Reply:
makes sense, will do